.github/workflows/build-and-deploy-staging.yaml

```
@@ -0,0 +1,79 @@
+name: Build
+on:
+  push:
+    branches:
+       - staging
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)